src/lib/evil/unposix/unistd.h

```diff
@@ -3,8 +3,6 @@
 
 #include "unimplemented.h"
 
-#include <evil_unistd.h>
-
```

Comment:
What this has to do with wrap dependency?

Reply:
Ditto.